src/client.ts

```diff
@@ -369,6 +467,29 @@ export class Client {
   // FILE MARKER - METHODS - PRIVATE ///////////////////////////////////////////
   //////////////////////////////////////////////////////////////////////////////
 
+  /**
+   * This method is used internally to calculate the element Viewport (Dimensions)
+   * @param selector
+   * @returns ViewPort object
+   */
+  private async getViewport(selector: string) {
+    const Values: DOMRect = JSON.parse(
```

Comment:
nitpick, but lowercase v for Values here please :P

Reply:
Yes with Priority xD